internal/cltest/cltest.go

```diff
@@ -226,6 +228,32 @@ func (ta *TestApplication) NewBox() packr.Box {
 	return packr.NewBox("../fixtures/gui/dist")
 }
 
+func (ta *TestApplication) StartAndConnect() error {
+	wg := sync.WaitGroup{}
+	wg.Add(1)
+	err := ta.Start()
+	if err != nil {
+		return err
+	}
+
+	ta.OnConnect(func() { // Invoked after Start to make sure it's the last callback.
+		wg.Done()
+	})
+
+	done := make(chan struct{}, 1)
+	go func() {
+		wg.Wait()
+		done <- struct{}{}
```

Comment:
Could we just have `ta.Start` send to `done` thereby eliminating the `WaitGroup`?